docs/upgrade/phpstan-level-1.md

```diff
@@ -0,0 +1,63 @@
+# Upgrade Instructions for Upgrading PHPStan to Level 1
+
+This article describes the upgrade instructions for [#1040 Upgrading PHPStan](https://github.com/shopsys/shopsys/pull/1040).
+Upgrade instructions are in a separate article because there is a lot of instructions and we don't want to jam the [UPGRADE-v7.3.0](/docs/upgrade/UPGRADE-v7.3.0.md).
+Instructions are meant to be followed when you upgrade from `v7.2.0` to `v7.3.0`.
+We recommend upgrading to level 1 to .
+
+To upgrade level of PHPStan you need to:
+- change the value of `phpstan.level` Phing property in your `build.xml` (`1` is the default value)
```

Comment:
```suggestion
- change the value of `phpstan.level` Phing property in your `build.xml`
```

as it is for me as for the reader confusing defaut value of what? phpstan? build.xml? And I think I don't need this information

Reply:
applied manually as a part of larger fixup